rules.adoc

```diff
@@ -775,7 +775,7 @@ ball.
 [[regulations-inference-in-lightweight]]
 ==== Infrared interference in Lightweight
 
-In Lightweight, the robot must not emit infrared light.
+Components designed to emit IR (e.g. IR time-of-flight LiDAR sensors) are not allowed and must be covered or removed if present at all. Any device designed to emit light of any wavelength must be mounted so that light is directed perpendicular to the playing field or be covered/removed.
```

Comment:
"must be covered or removed if present at all" could be read as ambiguous in terms of yes you can have it if you cover it. I'd like to suggest "[...] are not allowed and tournament organizers will require such devices to be removed or covered up".

Reply:
Sounds good - I'm just not sure how to make an edit to a pull request yet 😢

Reply:
You just push additional commits to the branch you created the pull request from - the PR requests to merge MikeRCJ:master into RoboCupJuniorTC:master and hence updates automatically. You'd also need to change the PR to request merging into the branch 2023-draft-rules.